### PR TITLE
Debug stable

### PR DESCRIPTION
**Pull request by carlos-buruel**
1. who is responsible for the release?
    Carlos Buruel Ortiz
2. Who reviewed the PR?
    Vidhya
3. What is the Git Tag number?
    1.1.0
4. What are the major changes for the release?
    - Change view for avoid bugs in Samnsung
    - Add support for pie chart to multi series
    - Integrate alert for webView when the limit row is visible
5. How will the new features be verified?
    They were reviewed by our testing.
6. What is the rollback procedure?
    Revert to versions 1.0.74
7. Is this change breaking?
    No

### CODE REVIEW

File: HTML/bar_series.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        data: [814, 841, 3714, 727, 31]//Verde
+    }, {
+        data: [1216, 1001, 4436, 738, 40]//Orange
+    }]
+		// series: 
+		// [
+		// 	{
+		// 		data: [["1", 107], ["2", 31], ["3", 635], ["4", 203], ["5", 2]]
+		// 	}
+		// ]
```
Comment (Ben-CHATA) [inline HTML/bar_series.html:109]: remove commented code if you don't need it

File: HTML/web_chata/js/donut.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			return 'translate(' + _d + ')';
+		})
+		.attr('dy', '.50em')
+		.style('text-anchor', 'middle')
+		.text(function(d) {
+			return d.data.name;
+			// if(d.data.percentage < 8) {
+			// 	return '';
+			// }
+			// return d.data.percentage + '%';
```
Comment (Ben-CHATA) [inline HTML/web_chata/js/donut.js:48]: remove this

File: HTML/web_chata/main_chart.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  /*border for svg*/
+  svg {
+    width: 99%;
+    position: relative;
+
+    border: 1px solid #aaa;
+    /*width: 100%;
+    position: relative;
+    height: 100%;
+    z-index: 0;*/
```
Comment (Ben-CHATA) [inline HTML/web_chata/main_chart.html:27]: remove

File: HTML/web_chata/stacked_bar.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  //   .enter().append('rect')
+  //     .attr('x', function(d) { return x(d.data.group); })
+  //     .attr('height', function(d) { return y(d[0]) - y(d[1]); })
+  //     .attr('y', function(d) { return y(d[1]); })
+  //     .attr('width', x.bandwidth())
+  //     .on('click', function(_, d) {
+  //       var subgroupName = d3.select(this.parentNode).datum().key;
+  //       var subgroupValue = d.data[subgroupName];
+  //       console.log('group: ' + subgroupName + ', value: ' + subgroupValue);
+  //     });
```
Comment (Ben-CHATA) [inline HTML/web_chata/stacked_bar.html:89]: remove unless you're still planning to work on this

File: app/src/main/res/layout/activity_bubble.xml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+	<!-- View at the top -->
+<!--	<FrameLayout-->
+<!--		android:background="@color/transparent"-->
+<!--		android:layout_height="match_parent"-->
+<!--		android:layout_width="match_parent">-->
+<!--		<TextView-->
+<!--			android:background="@color/blue_chata_circle"-->
+<!--			android:layout_width="64dp"-->
+<!--			android:layout_height="64dp"/>-->
+<!--	</FrameLayout>-->
```
Comment (Ben-CHATA) [inline app/src/main/res/layout/activity_bubble.xml:38]: remove?

File: app/src/main/res/layout/pager_activity.xml [hunk 6]
```diff
@@ -13,11 +14,12 @@
...
 	</androidx.viewpager.widget.ViewPager>
 
-	<chata.can.chata_ai.view.PagerOptions
-		android:id="@+id/pagerOption"
-		android:layout_width="match_parent"
-		android:layout_height="match_parent"/>
+<!--	<chata.can.chata_ai.view.PagerOptions1-->
+<!--		android:id="@+id/pagerOption"-->
+<!--		android:layout_width="match_parent"-->
+<!--		android:layout_height="match_parent"/>-->
```
Comment (Ben-CHATA) [inline app/src/main/res/layout/pager_activity.xml:24]: remove?

File: chata_ai/src/main/java/chata/can/chata_ai/dialog/twiceDrill/TwiceDrillPresenter.kt [hunk 7]
```diff
@@ -19,10 +19,10 @@ class TwiceDrillPresenter(
...
 	override fun onFailure(jsonObject: JSONObject?)
 	{
-		if (jsonObject != null)
-		{
-
-		}
+//		if (jsonObject != null)
+//		{
+//
+//		}
```
Comment (Ben-CHATA) [inline chata_ai/src/main/java/chata/can/chata_ai/dialog/twiceDrill/TwiceDrillPresenter.kt:25]: remove?

File: chata_ai/src/main/java/chata/can/chata_ai/extension/String.kt [hunk 8]
```diff
@@ -70,13 +72,11 @@ fun String.formatWithColumn(
...
 					{
 						SinglentonDrawer.localLocale?.let { locale ->
-							var date = ""
 							val dateFormat = SimpleDateFormat("yyyy-MM", locale)
 							try {
 								dateFormat.parse(this)?.let { dDate ->
-									val dateFormat2 = SimpleDateFormat(SinglentonDrawer.monthYearFormat, locale)
-//									val dateFormat2 = SimpleDateFormat("yyyy", locale)
-//									date = dateFormat2.format(dDate)
+//									val dateFormat2 = SimpleDateFormat(SinglentonDrawer.monthYearFormat, locale)
```
Comment (Ben-CHATA) [inline chata_ai/src/main/java/chata/can/chata_ai/extension/String.kt:78]: remove

File: chata_ai/src/main/java/chata/can/chata_ai/holder/BaseHolder.kt [hunk 9]
```diff
@@ -104,13 +108,18 @@ open class BaseHolder(
 
 						rlDelete?.visibility = View.VISIBLE
 						ivReport?.visibility =
-							if (item.message == "I want to make sure I understood your query. Did you mean:")
-								View.GONE
-							else View.VISIBLE
+//							if (item.message == "I want to make sure I understood your query. Did you mean:")
+//								View.GONE
+//							else
+//							{
```
Comment (Ben-CHATA) [inline chata_ai/src/main/java/chata/can/chata_ai/holder/BaseHolder.kt:114]: remove?

**Comment by carlos-buruel** [comment]
1. who is responsible for the release?
    Carlos Buruel Ortiz
2. Who reviewed the PR?
    Vidhya
3. What is the Git Tag number?
    1.1.0
4. What are the major changes for the release?
    - Change view for avoid bugs in Samnsung
    - Add support for pie chart to multi series
    - Integrate alert for webView when the limit row is visible
5. How will the new features be verified?
    They were reviewed by our testing.
6. What is the rollback procedure?
    Revert to versions 1.0.74
7. Is this change breaking?
    No

**Review by Ben-CHATA — APPROVED** [review]

**Pull request merged**